java/build-android.gradle

```diff
@@ -30,40 +48,113 @@ android {
 	buildToolsVersion "29.0.2"
 
 	defaultConfig {
-		minSdkVersion 24
+		minSdkVersion 21
 		targetSdkVersion 28
```

Comment:
How did we choose the minSdkVersion and targetSdkVersion? Comment capturing that would be good so we know the history if we consider changing those values.

Reply:
Moved this to params to gradle